src/helper_functions.js

```diff
@@ -0,0 +1,47 @@
+_ACTIVITIES = ["Versammlung", "Gastronomie","Lieferdienste", "Rausgehen",
```

Comment:
Would you mind putting everything in app.js?

That way, we don't have to modify all the templates.

Reply:
I didn't manage to test it from there, probably because my git setup is completely broken so I'm only submitting this by editing the files on github.

Can you pull it in like this and refactor next?

Reply:
Sure!